.bazelrc

```diff
@@ -60,8 +60,8 @@ build:asan --config=sanitizer
 # ASAN install its signal handler, disable ours so the stacktrace will be printed by ASAN
 build:asan --define signal_trace=disabled
 build:asan --define ENVOY_CONFIG_ASAN=1
-build:asan --copt -fsanitize=address
-build:asan --linkopt -fsanitize=address
+build:asan --copt -fsanitize=address,undefined
```

Comment:
Ugh, sorry, this is annoying: this particular difference is here because of https://github.com/envoyproxy/nighthawk/issues/593. I should have added a comment about that.

Reply:
Thanks, removed `undefined` and added. a comment.